test/Conversion/ExportVerilog/line-length.mlir

```diff
@@ -45,3 +45,18 @@ hw.module @longvariadic(%a: i8) -> (b: i8) {
 // LIMIT_LONG-NEXT:   wire [7:0] _tmp_0 = a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a + a +
 // LIMIT_LONG-NEXT:                       a + a + a + a + a + a + a + a + a;
 // LIMIT_LONG-NEXT:   assign b = _tmp + _tmp_0;
+
+hw.module @moduleWithComment()
+  attributes {comment = "The quick brown fox jumps over the lazy dog.  The quick brown fox jumps over the lazy dog."} {}
+
+// SHORT-LABEL:   // The quick brown fox jumps over the
+// SHORT-NEXT:    // lazy dog.  The quick brown fox jumps
+// SHORT-NEXT:    // over the lazy dog.
+// SHORT-NEXT:    module moduleWithComment
+//
+// DEFAULT-LABEL: // The quick brown fox jumps over the lazy dog.  The quick brown fox jumps over the lazy
+// DEFAULT-NEXT:  // dog.
+// DEFAULT-NEXT:  module moduleWithComment
+//
+// LONG-LABEL:    // The quick brown fox jumps over the lazy dog.  The quick brown fox jumps over the lazy dog.
+// LONG-NEXT:     module moduleWithComment
```

Comment:
Please add a test that has a really long string (200 characters or something) that cannot be broken.  Please also add a case that has embedded newlines to make sure they force a line break and are handled correctly.

Reply:
Added both of these. These comments now include an `a{200}` and show that that doesn't break.

I hadn't implemented the user-specified `\n` breaking, but that's added in the rebase. This is actually important as Grand Central relies on this behavior.